tests/suites/test_suite_psa_crypto.data

```diff
@@ -6981,27 +6981,27 @@ depends_on:PSA_WANT_ALG_CTR:PSA_WANT_KEY_TYPE_AES
 generate_key:PSA_KEY_TYPE_AES:64:PSA_KEY_USAGE_EXPORT | PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT:PSA_ALG_CTR:PSA_ERROR_INVALID_ARGUMENT:0
 
 PSA generate key: RSA, 512 bits, good, sign (PKCS#1 v1.5)
-depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:MBEDTLS_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_LEGACY:MBEDTLS_GENPRIME
+depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:MBEDTLS_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_LEGACY:MBEDTLS_GENPRIME:MBEDTLS_RSA_MIN_KEY_SIZE <= 512
```

Comment:
Since we're changing the default minimum, we should change the test cases. The test cases here cover two things:

* Variability over the algorithm used to exercise the key after generation (PKCS1v1.5 signature, PKCS1v1.5 encryption, PSS signature, OAEP encryption).
* Variability over the key size: internally, some things work differently depending on the key size mod 2, mod 8, mod 16.

I think that rather than add dependencies, we should use key sizes that work well with our default configuration. But still with dependencies so that the tests work in a configuration with a different minimum.